presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -77,6 +77,7 @@
         implements Closeable
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(OrcWriter.class).instanceSize();
+    private static final int LOGICAL_SIZE_MAX_SAMPLE_ROWS = 100;
```

Comment:
hmm... this constant name doesn't look easy to understand, but I don't really have a good suggestion.

What about just use 100 in the method. In general, constant instead of magic number is preferred, however, in this case: 
- It's very localized, and self-explained. 
- It's difficult to come up with a good constant name. 

which I think it's OK to use magic number inside `estimateAverageLogicalSizePerRow` .